src/framework/components/collision/system.js

```diff
@@ -155,6 +154,15 @@ class CollisionSystemImpl {
         }
     }
 
+    destroyShape(data) {
+        if (!data.shape) {
+            return;
+        }
+
+        Ammo.destroy(data.shape);
+        data.shape = null;
+    }
```

Comment:
```suggestion
    destroyShape(data) {
        if (data.shape) {
            Ammo.destroy(data.shape);
            data.shape = null;
        }
    }
```

Reply:
In fact this was to follow contributing guidelines on [early exiting](https://github.com/playcanvas/engine/blob/main/.github/CONTRIBUTING.md#exit-logic-early), are you sure this would really better?

Reply:
I understand where you're coming from, but for a short function like this, the early exit provides no advantages.

Reply:
Alright, it's committed then